modules/bullet/bullet_physics_server.cpp

```diff
@@ -235,9 +246,8 @@ RID BulletPhysicsServer3D::area_create() {
 	AreaBullet *area = bulletnew(AreaBullet);
 	area->set_collision_layer(1);
 	area->set_collision_mask(1);
-	CreateThenReturnRID(area_owner, area)
-}
-
+	CreateRIDReturn(area_owner, area);
+};
```

Comment:
Style bug, there's no need for this semicolon (and keeping the separation line after it would be nice).